sanic/worker/loader.py

```diff
@@ -109,18 +110,28 @@ class CertLoader:
     _creator_class: Type[CertCreator]
 
     def __init__(self, ssl_data: Dict[str, Union[str, os.PathLike]]):
+        self._ssl_data = ssl_data
+
         creator_name = ssl_data.get("creator")
-        if creator_name not in ("mkcert", "trustme"):
-            raise RuntimeError(f"Unknown certificate creator: {creator_name}")
-        elif creator_name == "mkcert":
-            self._creator_class = MkcertCreator
-        elif creator_name == "trustme":
-            self._creator_class = TrustmeCreator
+        if not creator_name:
+            return
+        else:
+            if creator_name not in ("mkcert", "trustme"):
+                raise RuntimeError(
+                    f"Unknown certificate creator: {creator_name}"
+                )
+            elif creator_name == "mkcert":
+                self._creator_class = MkcertCreator
+            elif creator_name == "trustme":
+                self._creator_class = TrustmeCreator
```

Comment:
Maybe use `dict(mkcert=MkcertCreator, trustme=TrustmeCreator).get(creator_name)` instead of this whole thing? Less code, no repetition.

Reply:
Was thinking about that, but was keeping my code changes minimal.